application/src/main/java/org/opentripplanner/transit/model/basic/Distance.java

```diff
@@ -36,6 +41,11 @@ public boolean equals(Object other) {
     }
   }
 
+  @Override
+  public int hashCode() {
+    return Objects.hash(meters);
```

Comment:
Based on @t2gran 's suggestion for the Ratio, this probably should be Integer.hashCode(meters).